app/models/application_record.rb

```diff
@@ -8,7 +8,11 @@ def self.fuzzy_search(term, **cols)
 
   def self.match_search(term, **cols)
     sanitized = sanitize_for_search term, **cols
-    select(Arel.sql("`#{table_name}`.*, #{sanitized} AS search_score")).where(sanitized)
+
+    mappedCols = sanitized.map { |val| "#{val} AS search_score_#{sanitized.find_index(val)}" }.join(', ')
+    whereClause = sanitized.map { |val| val.to_s }.join(' OR')
+
+    select(Arel.sql("`#{table_name}`.*, #{mappedCols}")).where(whereClause).order('search_score_0 * 2 + search_score_1 * 1 DESC')
```

Comment:
Wrap raw SQL statements in `Arel.sql` when you pass them to ActiveRecord:

```ruby
.order(Arel.sql('search_score_0 * 2 + search_score_1 * 1 DESC'))
```

Reply:
Do we need "* 1" for search_score_1?

Reply:
What does "* 1" do? Seems unnecessary - i.e., is that any different from just "search_score_0 * 2 + search_score 1 DESC"?

Reply:
Probably not - I just copied it out without really thinking about it :)

Reply:
Nope, does nothing really. Was just experimenting on how we would determine search relevancy